### PR TITLE
add html. header

### PR DESCRIPTION
**Pull request by Gkhaziuri**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```
Comment (SabaTsaava017) [inline index.html]: lesson 14 miweria me title shi

File: style.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.header {
+    height: 388px;
+    background-color: #0020F4;
+}
+.navigation {
+    display: inline-block;
```
Comment (Dmssnnn) [inline style.css:11]: navigaciashiis divshi ar aris sachiro inline-blockis gawera amjamad.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+.button {
+    background-color: white;
+    padding: 10px;
+    border-radius: 15px;
+    display: inline-block;
+    border: solid white;
+}
+ul {
```
Comment (Dmssnnn) [inline style.css:36]: no need.  " li " shi gaqvs gawerili inline-block

File: style.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    display: inline-block;
+}
+.header-container {
+    text-align: center;
+    margin: 0 auto;
+}
```
Comment (saba-ab) [inline style.css]: kargia mami shebere

File: style.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.header {
+    height: 388px;
```
Comment (saba-ab) [inline style.css:7]: simagle amas ar gauwerooo gauwere apddingiii

**Review by saba-ab — COMMENTED** [review]
midi mami

**Pull request merged**